config/l3afd.cfg

```diff
@@ -14,33 +14,25 @@ bpf-chaining-enabled: true
 swagger-api-enabled: false
 # PROD | DEV
 environment: PROD
-#BpfMapDefaultPath is base path for storing maps 
+# BpfMapDefaultPath is base path for storing maps
 BpfMapDefaultPath: /sys/fs/bpf
 
 
 [kf-repo]
-url:
+url: http://localhost:8000/
 
 [web]
 metrics-addr: 0.0.0.0:8898
 kf-poll-interval: 30s
 n-metric-samples: 20
 
-[admind]
-host: 
-username: 
-api-key: 
-group-id: 0
-api-enabled: true
-
-
 [xdp-root-program]
 name: xdp-root
 artifact: xdp-root.tar.gz
 ingress-map-name: root_array
 command: xdp_root
 version: 1.01
-is-user-program: false
+user-program-daemon: false
```

Comment:
This is a big confusing, as I don't think we have any user-mode program associated with the root program.  Should we always assume this is false for root chaining programs?

Reply:
yes agree, we can remove this entry.